packages/dds/sequence-sequential-id/src/generateSequentialId.ts

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+
+export const digitZero = "A";
+export const digitLast = "Z";
+const idBase = 26;
+
+export function isValidSeqId(seqId: string): boolean {
+  for (let i = 1; i < seqId.length; i++) {
+    const letterCode = getLetterCode(seqId[i]);
+    if (letterCode < 0 && letterCode >= idBase) {
+      return false;
+    }
+  }
+  return true;
+}
+
+const nextChar = (char: string) => String.fromCharCode(char.charCodeAt(0) + 1);
+
+// Returns value for UpperCase letter from 0 - 25;
+const getLetterCode = (char: string) => char.charCodeAt(0) - digitZero.charCodeAt(0);
+
+export function createIdBetween(seqId1: string, seqId2: string): string {
+  const delta: number[] = [];
+  let minSeqId = seqId1.length > 0 ? seqId1 : digitZero;
+  let maxSeqId = seqId2.length > 0 ? seqId2 : digitLast;
+  if (minSeqId.length !== maxSeqId.length) {
```

Comment:
it would be great to assert that minSeqId <= maxSeqId as a pre-condition for this function (unless we do not care, in which case I'd rename variables to remove min & max references)

Reply:
We do care! Let me add an assert for this!

Reply:
Added the assert for this